hack/make.ps1

```diff
@@ -94,6 +94,7 @@ param(
 $ErrorActionPreference = "Stop"
 $ProgressPreference = "SilentlyContinue"
 $pushed=$False  # To restore the directory if we have temporarily pushed to one.
+Set-Variable GOTESTSUM_LOCATION -option Constant -value "$env:GOPATH/src/gotest.tools/gotestsum"
```

Comment:
If we compile `gotestsum` to be put in `$GOBIN` (which defaults to `$GOPATH\bin`), we don't need this variable, and we can just call `gotestsum.exe ...`

If would mean we have to keep track of less things, and can simplify the scripts

Reply:
Actually this will require a separate copy to $GOBIN and introducing or at least using a new variable $GOBIN which is currently not being used anywhere in these scripts. And it will save us from having to track things in only one place. So I am afraid it will actually end up increasing complexity rather than decreasing it.